pkg/columns/formatter/json/json.go

```diff
@@ -46,7 +47,12 @@ func NewFormatter[T any](cols columns.ColumnMap[T], options ...Option) *Formatte
 
 	ncols := make([]*column[T], 0)
 	for _, col := range cols.GetOrderedColumns() {
-		name, _ := json.Marshal(col.Name)
+		colName := col.Name
+		if strings.Contains(colName, ".") {
+			hierarchy := strings.Split(colName, ".")
+			colName = hierarchy[len(hierarchy)-1]
+		}
+		name, _ := json.Marshal(colName)
```

Comment:
What about columns having a `a.b.c` name, will those be supported?

Reply:
Yes, this is done recursively. For support see the `TestJSONFormatter` test which has a `parent.child1.grandchild1` test case and more